src/simple_server/simple_game.clj

```diff
@@ -1,23 +1,30 @@
 (ns simple-server.simple-game)
 
-
-(def game-in-progress (atom nil))
+;; Creating a map for a game, consisting of the number to guess and the number of guesses remains
+(def game-in-progress (atom {:number nil
```

Comment:
How are multiple independent users going to play the game simultaneously?

Reply:
This has not been implemented, unfortunately...